toolkit/scripts/toolchain/container/toolchain-remote-wget-list

```diff
@@ -64,71 +64,6 @@ https://www.openssl.org/source/openssl-1.1.1g.tar.gz
 https://ftp.gnu.org/gnu/sed/sed-4.8.tar.xz
 https://www.sqlite.org/2020/sqlite-autoconf-3320100.tar.gz
 https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.37/util-linux-2.37.2.tar.xz
-https://www.x.org/pub/individual/lib/xtrans-1.3.5.tar.bz2
-https://www.x.org/pub/individual/lib/libX11-1.6.5.tar.bz2
-https://www.x.org/pub/individual/lib/libXext-1.3.3.tar.bz2
-https://www.x.org/pub/individual/lib/libFS-1.0.7.tar.bz2
-https://www.x.org/pub/individual/lib/libICE-1.0.9.tar.bz2
-https://www.x.org/pub/individual/lib/libSM-1.2.2.tar.bz2
-https://www.x.org/pub/individual/lib/libXScrnSaver-1.2.2.tar.bz2
-https://www.x.org/pub/individual/lib/libXt-1.1.5.tar.bz2
-https://www.x.org/pub/individual/lib/libXmu-1.1.2.tar.bz2
-https://www.x.org/pub/individual/lib/libXpm-3.5.12.tar.bz2
-https://www.x.org/pub/individual/lib/libXaw-1.0.13.tar.bz2
-https://www.x.org/pub/individual/lib/libXfixes-5.0.3.tar.bz2
-https://www.x.org/pub/individual/lib/libXcomposite-0.4.4.tar.bz2
-https://www.x.org/pub/individual/lib/libXrender-0.9.10.tar.bz2
-https://www.x.org/pub/individual/lib/libXcursor-1.1.14.tar.bz2
-https://www.x.org/pub/individual/lib/libXdamage-1.1.4.tar.bz2
-https://www.x.org/pub/individual/lib/libfontenc-1.1.3.tar.bz2
-https://www.x.org/pub/individual/lib/libXfont2-2.0.1.tar.bz2
-https://www.x.org/pub/individual/lib/libXft-2.3.2.tar.bz2
-https://www.x.org/pub/individual/lib/libXi-1.7.9.tar.bz2
-https://www.x.org/pub/individual/lib/libXinerama-1.1.3.tar.bz2
-https://www.x.org/pub/individual/lib/libXrandr-1.5.1.tar.bz2
-https://www.x.org/pub/individual/lib/libXres-1.0.7.tar.bz2
-https://www.x.org/pub/individual/lib/libXtst-1.2.3.tar.bz2
-https://www.x.org/pub/individual/lib/libXv-1.0.11.tar.bz2
-https://www.x.org/pub/individual/lib/libXvMC-1.0.10.tar.bz2
-https://www.x.org/pub/individual/lib/libXxf86dga-1.1.4.tar.bz2
-https://www.x.org/pub/individual/lib/libXxf86vm-1.1.4.tar.bz2
-https://www.x.org/pub/individual/lib/libdmx-1.1.3.tar.bz2
-https://www.x.org/pub/individual/lib/libpciaccess-0.13.5.tar.bz2
-https://www.x.org/pub/individual/lib/libxkbfile-1.0.9.tar.bz2
-https://www.x.org/pub/individual/lib/libxshmfence-1.2.tar.bz2
 https://download.savannah.gnu.org/releases/freetype/freetype-2.9.1.tar.gz
```

Comment:
who needs fontconfig and freetype?  Can those be removed too?

Reply:
No, fontconfig and freetype cannot be removed. There are a number of packages that still depend on them (cairo, harfbuzz, qt4-qtbase, pango, etc..)

Reply:
Actually, you're right that these can be removed from the toolchain, since jdk8 was the only toolchain component that needed them. We just need to keep the specs for other package dependencies